<launch>

  <!--value值确定频率-->
  <param name="/freq" type="double" value="1000"></param>
  <node pkg="ros_franka" type="Frequency_Control" name="FrankaFrequency" output="screen"/>
  <node pkg="ros_franka" type="ForceAndPos_Control" name="ForceAndPos_Control" output="screen"/>

</launch>